<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleConfig</class>
 <widget class="QWidget" name="StyleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>344</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Widget Style</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Qt Style</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="qtComboBox"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="paletteGroupBox">
         <property name="title">
          <string>Qt Palette</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Some Qt styles may ignore these colors.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QFormLayout" name="formLayout_4">
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="verticalSpacing">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Window:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="ColorLabel" name="winColorLabel"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>View:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="ColorLabel" name="baseColorLabel"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Selection:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="ColorLabel" name="highlightColorLabel"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Link:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="ColorLabel" name="linkColorLabel"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_5">
              <property name="formAlignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="verticalSpacing">
               <number>5</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Window Text:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="ColorLabel" name="windowTextColorLabel"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>View Text:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="ColorLabel" name="viewTextColorLabel"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Selected Text:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="ColorLabel" name="highlightedTextColorLabel"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Visited Link:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="ColorLabel" name="linkVisitedColorLabel"/>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QToolButton" name="savePaletteBtn">
              <property name="text">
               <string>&amp;Save Palette</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="loadPaletteBtn">
              <property name="text">
               <string>&amp;Load Palette</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>5</width>
                <height>5</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="defaultPaletteBtn">
              <property name="text">
               <string>&amp;Default Palette</string>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextOnly</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Toolbar button style:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="toolButtonStyle">
         <item>
          <property name="text">
           <string>Only display the icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Only display the text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The text appears beside the icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>The text appears under the icon</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QCheckBox" name="singleClickActivate">
         <property name="text">
          <string>Activate item on single click</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorLabel</class>
   <extends>QLabel</extends>
   <header>colorLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
